test/kubemark/start-kubemark.sh

```diff
@@ -184,7 +189,11 @@ function create-kube-hollow-node-resources {
   # Create the replication controller for hollow-nodes.
   # We allow to override the NUM_REPLICAS when running Cluster Autoscaler.
   NUM_REPLICAS=${NUM_REPLICAS:-${NUM_NODES}}
-  sed "s@{{numreplicas}}@${NUM_REPLICAS}@g" "${RESOURCE_DIRECTORY}/hollow-node_template.yaml" > "${RESOURCE_DIRECTORY}/hollow-node.yaml"
+  if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
+    sed "s@{{numreplicas}}@${NUM_REPLICAS}@g" "${RESOURCE_DIRECTORY}/hollow-node_template_scaleout.yaml" > "${RESOURCE_DIRECTORY}/hollow-node.yaml"
```

Comment:
it looks the diffs from the yaml files for scaleout an non-scaleout is not much, should we maintain one yaml file for them instead of creating two? consider this will be eventually into the main branch ? i am ok with two, just something to make sure we evaluated the long term pros-cons

Reply:
let's do it when we fix the kubemark regression.